drivers/theora/theora.h

```diff
@@ -259,7 +259,7 @@ typedef struct{
  * This structure holds the in-stream metadata corresponding to
  * the 'comment' header packet.
  *
- * Meta data is stored as a series of (tag, value) pairs, in
+ * Command data is stored as a series of (tag, value) pairs, in
```

Comment:
That example you just pointed out in the code is fair, and I didn't realize that the Linux key was called Meta. I'm gonna close this PR because it doesn't make sense to change it then. (It didn't change KEY_META to KEY_COMMAND though) My mistake, sorry guys.